src/server/app.js

```diff
@@ -46,17 +46,17 @@ app.use((err, req, res) => {
     }
     return res.send({ error: err.message });
   }
-  res.sendStatus(500);
+  // res.sendStatus(500);
 });
 
-app.use('/api/', function (req, res) {
-  res.status(404).send("Sorry can't find that!");
-});
+// app.use('/api/', function (req, res) {
+//   res.status(404).send("Sorry can't find that!");
+// });
 
 // If "/api" is called, redirect to the API documentation.
-app.use('/api', function (req, res) {
-  res.redirect(`${process.env.API_PATH}/documentation`);
-});
+// app.use('/api', function (req, res) {
+//   res.redirect(`${process.env.API_PATH}/documentation`);
+// });
```

Comment:
Can you please undo changes in this file?

Reply:
Yes I will